libethereum/EthereumHost.cpp

```diff
@@ -640,7 +640,9 @@ shared_ptr<Capability> EthereumHost::newPeerCapability(shared_ptr<SessionFace> c
 {
 	auto ret = HostCapability<EthereumPeer>::newPeerCapability(_s, _idOffset, _cap, _capID);
 
-	capabilityFromSession<EthereumPeer>(*_s)->init(
```

Comment:
We requested here the new capability just registered on the previous line, but forgot to specify its version.